druntime/src/core/demangle.d

```diff
@@ -3273,3 +3273,13 @@ private struct BufSlice
     auto getSlice() inout nothrow scope { return buf[from .. to]; }
     size_t length() const scope { return to - from; }
 }
+
+extern(C) export int __d_demangle(const(char*) mangled, char* buffer, size_t bufferLength)
```

Comment:
Double underscore identifiers are reserved for names the compiler generates, don't use it for functions you expect a user to call.